rbac/internal/views.py

```
@@ -265,6 +266,71 @@ def tenant_view(request, org_id):
+            return JsonResponse({"error": str(e)}, status=400)
+
+        old_limit = tenant.workspace_creation_limit()
+        tenant.org_config = new_config
+        try:
+            tenant.save(update_fields=["org_config"])
+        except ValidationError as e:
+            return JsonResponse({"error": Tenant.org_config_error_message(e)}, status=400)
+
+        try:
+            TENANTS.delete_tenant(org_id)
+        except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)